src/Illuminate/Support/Str.php

```diff
@@ -1220,40 +1217,25 @@ public static function wordCount($string, $characters = null)
     }
 
     /**
-     * Generate a UUID (version 4).
+     * Generate a UUID (version 7).
      *
      * @return \Ramsey\Uuid\UuidInterface
      */
     public static function uuid()
     {
         return static::$uuidFactory
                     ? call_user_func(static::$uuidFactory)
-                    : Uuid::uuid4();
+                    : Uuid::uuid7();
     }
 
     /**
-     * Generate a time-ordered UUID (version 4).
+     * Generate a time-ordered UUID (version 7).
      *
      * @return \Ramsey\Uuid\UuidInterface
      */
     public static function orderedUuid()
```

Comment:
What do you think about deprecation of this method?

Reply:
We specifically decided against it and will keep it as an alias indefinitely to prevent apps and packages from having to adjust to it.